fixtures/component_fixtures/common/EC2UnsupportedVersion/component.yaml

```diff
@@ -14,7 +14,7 @@ satisfies:
   implementation_status: partial
   narrative: Justification in narrative form
   standard_key: NIST-800-53
-schema_version: 1.0
+schema_version: 1.0.0
```

Comment:
Oh, didn't see this file is `Unsupported`...well, will it still fail similarly gracefully?